docs/LOOKUP_ARCHITECTURE.md

```
@@ -0,0 +1,650 @@
+| `LookupIntegrationService` | Integration with external systems |
+| `WorkflowIntegration` | Integration with workflow execution |
+
+---
+
+## API Endpoints
+
+Base URL: `/api/v2/unstract/{org_id}/lookup/`
+
+### Project Management
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)